pkg/util/file.go

```diff
@@ -14,6 +16,19 @@ func SetFileModeForPath(name string, mode os.FileMode) error {
 	return os.Chmod(name, mode)
 }
 
+func SetFileGroupForPath(name string, group string) error {
+	g, err := user.LookupGroup(group)
```

Comment:
What if the user provides a numeric group ID? We should support both names and IDs.

Reply:
Hmm, I had a suspicion so I checked this and from what I'm reading, group names that purely consist of digits are valid names in POSIX and also in various Linux distros, and utilities seem to be intended to treat strings of digits as just a string, not try to magically lookup.

https://unix.stackexchange.com/a/11481/408204

Though, I'm all for giving users choice (thus the issue in the first place)! Any ideas on a good ui to enable this without doing too much automagic?